src/main/java/aquality/selenium/elements/ElementFactory.java

```diff
@@ -39,6 +39,7 @@ public ElementFactory(IConditionalWait conditionalWait, IElementFinder elementFi
         typesMap.put(IButton.class, Button.class);
         typesMap.put(ICheckBox.class, CheckBox.class);
         typesMap.put(IComboBox.class, ComboBox.class);
+        typesMap.put(IMultiChoiceComboBox.class, MultiChoiceComboBox.class);
```

Comment:
maybe simple IMultiChoiceBox? Or it is necessary to emphasize that it is combobox?

Reply:
Naming is definitely up for discussion

Reply:
+1 vote for IMultiChoiseBox